lasagne/layers/noise.py

```diff
@@ -14,7 +14,7 @@
 
 
 class DropoutLayer(Layer):
-    """Dropout layer [1]_,[2]_.
+    """Dropout layer [1]_, [2]_
```

Comment:
Interesting -- was it the extra blank or the removed dot that fixed the reference?